packages/utils/test/abi_encoder/abi_samples/constructor_abis.ts

```diff
@@ -0,0 +1,24 @@
+import { ConstructorAbi } from 'ethereum-types';
+
+export const simpleAbi: ConstructorAbi = {
+    inputs: [
+        {
+            name: 'greg',
```

Comment:
Ah good, so I'm not the only one doing this :')

Reply:
I just copied yours :)